docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/LightGbm.cs

```diff
@@ -20,10 +20,10 @@ public static void Example()
 
             // Create the Estimator pipeline. For simplicity, we will train a small tree with 4 leaves and 2 boosting iterations.
             var pipeline = mlContext.Ranking.Trainers.LightGbm(
-                numLeaves: 4,
-                minDataPerLeaf: 10,
+                leafCount: 4,
```

Comment:
```suggestion
                numberOfLeaves: 4,
```
as suggested in #2613.